aspnetcore/security/authentication/windowsauth.md

```diff
@@ -80,117 +97,94 @@ Enable the IIS Role Service for Windows Authentication. For more information, se
-
 Use **either** of the following approaches:
 
-* [Development-side configuration before publishing the app](#development-side-configuration-with-a-local-webconfig-file) (*Recommended*)
-* [Server-side configuration after publishing the app](#server-side-configuration-with-the-iis-manager)
+* **Before publishing and deploying the project,** add the following *web.config* file to the project root:
 
-#### Development-side configuration with a local web.config file
+  [!code-xml[](windowsauth/sample_snapshot/web_2.config)]
 
-Perform the following steps **before** you [publish and deploy your project](#publish-and-deploy-your-project-to-the-iis-site-folder).
+  When the project is published by the SDK (without the `<IsTransformWebConfigDisabled>` property set to `true` in the project file), the published *web.config* file includes the `<location><system.webServer><security><authentication>` section. For more information on the `<IsTransformWebConfigDisabled>` property, see <xref:host-and-deploy/iis/index#webconfig-file>.
 
-Add the following *web.config* file to the project root:
+* **After publishing and deploying the project,** perform server-side configuration with the IIS Manager:
 
-[!code-xml[](windowsauth/sample_snapshot/web_2.config)]
+  1. In IIS Manager, select the IIS site under the **Sites** node of the **Connections** sidebar.
+  1. Double-click **Authentication** in the **IIS** area.
+  1. Select **Anonymous Authentication**. Select **Disable** in the **Actions** sidebar.
+  1. Select **Windows Authentication**. Select **Enable** in the **Actions** sidebar.
 
-When the project is published by the SDK (without the `<IsTransformWebConfigDisabled>` property set to `true` in the project file), the published *web.config* file includes the `<location><system.webServer><security><authentication>` section. For more information on the `<IsTransformWebConfigDisabled>` property, see <xref:host-and-deploy/iis/index#webconfig-file>.
+  When these actions are taken, IIS Manager modifies the app's *web.config* file. A `<system.webServer><security><authentication>` node is added with updated settings for `anonymousAuthentication` and `windowsAuthentication`:
 
-#### Server-side configuration with the IIS Manager
+  [!code-xml[](windowsauth/sample_snapshot/web_1.config?highlight=4-5)]
 
-Perform the following steps **after** you [publish and deploy your project](#publish-and-deploy-your-project-to-the-iis-site-folder).
+  The `<system.webServer>` section added to the *web.config* file by IIS Manager is outside of the app's `<location>` section added by the .NET Core SDK when the app is published. Because the section is added outside of the `<location>` node, the settings are inherited by any [sub-apps](xref:host-and-deploy/iis/index#sub-applications) to the current app. To prevent inheritance, move the added `<security>` section inside of the `<location><system.webServer>` section that the SDK provided.
```

Comment:
```suggestion
  The `<system.webServer>` section added to the *web.config* file by IIS Manager is outside of the app's `<location>` section added by the .NET Core SDK when the app is published. Because the section is added outside of the `<location>` node, the settings are inherited by any [sub-apps](xref:host-and-deploy/iis/index#sub-applications) to the current app. To prevent inheritance, move the added `<security>` section inside of the `<location><system.webServer>` section that the .NET Core SDK provided.
```